tycho-core/src/test/java/org/eclipse/tycho/core/ee/StandardExecutionEnvironmentTest.java

```diff
@@ -27,9 +27,7 @@
 
 public class StandardExecutionEnvironmentTest {
 
-    private StandardExecutionEnvironment javaSE18Environment;
-    private StandardExecutionEnvironment javaSE17Environment;
-    private StandardExecutionEnvironment javaSE11Environment;
+    private StandardExecutionEnvironment runningJavaEnvironment;
     private StandardExecutionEnvironment javaSE9Environment;
     private StandardExecutionEnvironment javaSE8Environment;
     private StandardExecutionEnvironment javaSE7Enviroment;
```

Comment:
you could do me a favor by search/replacing the typo "enviroment" (missing n) :)

Reply:
Done